db/seeds.rb

```diff
@@ -5,18 +5,19 @@ def initialize
 
   def urls
     ["http://apple.com",
-      "https://apple.com",
-      "https://www.apple.com",
-      "http://developer.apple.com",
-      "http://en.wikipedia.org",
-      "http://opensource.org"]
+     "https://apple.com",
+     "https://www.apple.com",
+     "http://developer.apple.com"
+      ]
   end
 
   def referrers
     ["http://apple.com",
-     "https://apple.com",
-     "https://www.apple.com",
-     "http://developer.apple.com",
+    "https://apple.com",
+    "https://www.apple.com",
+    "http://developer.apple.com",
+    "http://en.wikipedia.org",
+    "http://opensource.org",
      nil]
   end
```

Comment:
In the exercise description, it had the referrers and urls swapped, but I am assuming it was meant this way since we we would only want to track Apple's urls and not an outside url like opensource.org